challenges/objects-arrays.js

```diff
@@ -55,11 +88,24 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i=0; i<graduates.length; i++){
+  contactInfo.push(graduates[i]["first_name"]);
+  contactInfo.push(graduates[i]["email"]);
+  
+
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i = 0; i<graduates.length; i++){
+  if (graduates.includes(graduates[i].university) === "Uni"){
```

Comment:
You are on the right track
if(graduates[i].university.includes("Uni")) {
    uni.push(graduates[i].university); //?